### PR TITLE
[Configuration] Add dev_mode_enabled

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

File: octobot_commons/configuration/configuration.py [hunk 1]
```diff
@@ -289,3 +289,7 @@ def _get_config_without_profile_elements(self) -> dict:
         for profile_managed_element in self.profile.FULLY_MANAGED_ELEMENTS:
             filtered_config.pop(profile_managed_element, None)
         return filtered_config
+
+    def dev_mode_enabled(self) -> bool:
+        return os.getenv(commons_constants.CONFIG_DEBUG_OPTION,
+                         self.config.get(commons_constants.CONFIG_ACCEPTED_TERMS, False))
```
Comment (GuillaumeDSM) [inline octobot_commons/configuration/configuration.py:295]: `self.config.get(commons_constants.CONFIG_ACCEPTED_TERMS, False)` ? ^^

Comment (valouvaliavlo) [reply octobot_commons/configuration/configuration.py]: wtf ^^

**Review by GuillaumeDSM — APPROVED** [review]

File: octobot_commons/configuration/configuration.py [hunk 2]
```diff
@@ -33,21 +33,21 @@ class Configuration:
     """
 
     def __init__(
-        self,
-        config_path: str,
-        profiles_path: str,
-        schema_path: str = None,
-        profile_schema_path: str = None,
+            self,
```
Comment (Herklos) [inline octobot_commons/configuration/configuration.py:36]: black wont like it

File: octobot_commons/configuration/configuration.py [hunk 3]
```diff
@@ -289,3 +289,7 @@ def _get_config_without_profile_elements(self) -> dict:
         for profile_managed_element in self.profile.FULLY_MANAGED_ELEMENTS:
             filtered_config.pop(profile_managed_element, None)
         return filtered_config
+
+    def dev_mode_enabled(self) -> bool:
```
Comment (Herklos) [inline octobot_commons/configuration/configuration.py:293]: :+1:

**Review by Herklos — CHANGES_REQUESTED** [review]

File: octobot_commons/configuration/configuration.py (hunk 2 above)
Comment (valouvaliavlo) [reply octobot_commons/configuration/configuration.py]: sorry, pycharm do it automatically. it's fixed now

**Review by GuillaumeDSM — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Pull request merged**